definitions/npm/marked-man_v0.1.x/flow_v0.28.x-/marked-man_v0.1.x.js

```diff
@@ -0,0 +1,64 @@
+import marked from 'marked'
+
+type NodeCallback<T> = (e: ?Error, d: ?T) => void
```

Comment:
Can you move these into the body of the `declare module`? As-is, these create global types